app/collections/preview.cjsx

```diff
@@ -28,17 +28,18 @@ module.exports = React.createClass
       @setState {subjects}
 
   render: ->
-    [owner, name] = @props.collection.slug.split('/')
-
+    baseLink = ""
+    if @props.project?
+      baseLink += "projects/#{@props.project.slug}"
     <div className="collection-preview">
       <div className="collection">
         <p className="title">
-          <Link to="/collections/#{owner}/#{name}">
+          <Link to="#{baseLink}/collections/#{@props.collection.slug}">
```

Comment:
Sorry, yeah, I switched all the `baseLink`s from `"/"` to `""` per an earlier code review comment, looks like I missed the `/` from the project version of `baseLink` in this case. Well spotted.